src/components/ui/Chat/ChannelList/ChannelItem.tsx

```diff
@@ -22,25 +22,79 @@ export interface ChannelItemProps
   isSelected?: boolean;
   /** Callback function when clicked */
   onClick: () => void;
-  /** Signifies of there are unread message in the channel. */
+  /** Signifies if there are unread message in the channel. */
   unread?: boolean;
   /** Content of the badge signaling unread messages, such as a badge displaying the number. */
   unreadBadgeLabel?: string;
+  /* Content of the last channel message to be displayed */
+  lastChannelMessage?: string;
+  /* Timestamp of the last channel message to be displayed */
+  lastChannelMessageTimestamp?: string;
 }
 
 export const ChannelItem: FC<ChannelItemProps> = (props) => {
-  const { name, actions, isSelected, onClick, unread, unreadBadgeLabel } =
-    props;
+  const {
+    name,
+    actions,
+    isSelected,
+    onClick,
+    unread,
+    unreadBadgeLabel,
+    lastChannelMessage,
+    lastChannelMessageTimestamp,
+  } = props;
+
+  const displayDetailedChannelView =
```

Comment:
Can we jsut do `displayDetailedView` here

Reply:
Ah yes can rename this.